jest.setup.js

```diff
@@ -0,0 +1 @@
+delete process.env.DEFAULT_SITE_KEY;
```

Comment:
This probably isn't relevant (and therefore the file). Can't see that it's being read anywhere?